src/index.ts

```diff
@@ -1,7 +1,79 @@
+import { euclidean } from 'ml-distance-euclidean';
 /**
- * My module
- * @returns A very important number
+ * DBSCAN
+ * @returns list of cluster，-1 is noise
  */
-export function myModule(): number {
-  return 42;
+interface IPoint {
+  index: number;
+  value: number[];
+  label: number;
+}
+function rangeQuery(
+  cur: IPoint,
```

Comment:
In general we avoid abbreviation so that the code is relatively easy to read. I guess cur is `current`

Reply:
ok